dynamiqs/result.py

```diff
@@ -208,63 +213,45 @@ def clicktimes(self) -> Array:
     def nclicks(self) -> Array:
         return jnp.count_nonzero(~jnp.isnan(self.clicktimes), axis=-1)
 
-    @property
-    def final_state_norm(self) -> Array:
-        return self._saved.final_state_norm
-
-    def noclick_states(self) -> QArray:
-        return self.states[..., 0, :, :, :]
-
-    @property
-    def noclick_prob(self) -> Array:
-        # todo: fix
-        return None
-
     def _str_parts(self) -> dict[str, str | None]:
         d = super()._str_parts()
-        return d | {
-            'Clicktimes': _array_str(self.clicktimes),
-            'Number of clicks': _array_str(self.nclicks),
-            'No click probability': _array_str(self.noclick_prob),
-        }
-
-    @classmethod
-    def out_axes(cls) -> SolveResult:
-        return cls(None, None, None, None, 0, 0, 0)
+        return d | {'Clicktimes': _array_str(self.clicktimes)}
 
-    @property
     def mean_states(self) -> QArray:
```

Comment:
There is a mismatch between these two things. Either it's a property and it can be called `mean_states`, or it's a method and it should be something like `compute_mean_states` (but I find that verbose).